File: Triangle/Triangle.cpp

#include"Triangle.h"


Triangle::Triangle()
{
	l=1;
	b=1;

}

Triangle::Triangle(int len,int bre)
{
	l=len;
	b=bre;

}

Triangle::~Triangle()
{

}

void Triangle::CalculateArea(int le,int br)
{
	float Area=((le*br)/2);
	printf("area is %f \n",Area);
}

void Triangle::CalculateArea()
{
	float AreaT=((l*b)/2);
	printf("area is %f\n",AreaT);

}
	

File: shri-cpp-aug-19/second/assignment/Java/PolymorphismDemo/src/training/demo/model/Triangle.java
package training.demo.model;

public class Triangle extends ShapeImpl {
	private int height,s1,s2,base;
	public Triangle() {
		height=0;
		s1=0;
		s2=0;
		base=0;
	}
	public Triangle(int h,int b,int s1,int s2) {
		height=h;
		this.s1=s1;
		this.s2=s2;
		base=b;
	}
	public float area() {
		 return 0.5f*base*height;
	}
	public float perimeter() {
		 return base+s1+s2;
	}

}
